packages/portfolio-contract/src/resolver/resolver.exo.ts

```diff
@@ -137,6 +170,40 @@ export const prepareResolverKit = (
     }),
     {
       client: {
+        createPendingTx(txMeta: TxMeta) {
```

Comment:
How is this different from `registerTx`? I suggest some doc to say how.

Should we `@deprecate` `registerTx`?